Application/Api/Education/Certificate/Generator/Repository/ESBD/EsbdMsHjInfoFsGeistigeEntwicklung.php

```diff
@@ -0,0 +1,88 @@
+<?php
+
+namespace SPHERE\Application\Api\Education\Certificate\Generator\Repository\ESBD;
+
+use SPHERE\Application\Education\Certificate\Generator\Repository\Element;
+use SPHERE\Application\Education\Certificate\Generator\Repository\Page;
+use SPHERE\Application\Education\Certificate\Generator\Repository\Slice;
+use SPHERE\Application\People\Person\Service\Entity\TblPerson;
+
+/**
+ * Class EsbdMsHjInfoFsGeistigeEntwicklung
+ *
+ * @package SPHERE\Application\Api\Education\Certificate\Generator\Repository\ESBD
+ */
+class EsbdMsHjInfoFsGeistigeEntwicklung extends EsbdStyle
+{
+    /**
+     * @param TblPerson|null $tblPerson
+     *
+     * @return Page[]
+     */
+    public function buildPages(TblPerson $tblPerson = null)
+    {
+
+        $personId = $tblPerson ? $tblPerson->getId() : 0;
+
+        $pageList[] = $this->getPageOne($personId);
+        $pageList[] = $this->getPageTwo($personId);
+
+        return $pageList;
+    }
+
+    /**
+     * @param $personId
+     *
+     * @param string $title
+     * @param bool $isSignPartExtended
+     * @param string $term
+     * @return Page
+     */
+    public function getPageOne(
+        $personId,
+        $title = 'Halbjahresinformation der Oberschule',
```

Comment:
Codacy found an issue: [Incorrect spacing between argument "$title" and equals sign; expected 0 but found 1](https://app.codacy.com/manual/gerdchristian-kunze/SPHERE-Framework/pullRequest?prid=5757982)